core/service/src/engine/backup_operator.rs

```diff
@@ -658,6 +726,53 @@ where
     Ok(())
 }
 
+async fn update_specific_backup_vault_for_all_epochs<
```

Comment:
Does this mean that we update the backups after each epoch change such that the most recent backup is of shares based on the latest epoch?

In that case I believe we also need to ensure backups under old epoch ids get removed whenever this method is called